### PR TITLE
Feat/pan eu migration

### PR DESCRIPTION
**Pull request by persas**
Closes to #2009

### CODE REVIEW

File: src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts [hunk 1]
```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/no-unresolved
```
Comment (minotogna) [inline src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts]: This doesn’t look good , maybe better to rename the component itself in upper case ?

File: src/i18n/resources/en.js [hunk 2]
```diff
@@ -2010,5 +2010,257 @@ The FRA team
       no19_area_available_for_public_recreation: 'Area available for public recreation 19',
       no20_area_available_for_public_recreation: 'Area available for public recreation 20',
     },
+
+    // Country Comments
+    countryComments: {
```
Comment (minotogna) [inline src/i18n/resources/en.js]: Better to add country comments translations under panEuropean file with prefix panEuropean and import them from countryComments file

File: src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts (hunk 1 above)
Comment (persas) [reply src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts]: Yes, I don't know why the precommit hook was complaining about this. I haven't created this, I didn't want to modify it. But will do it.

Comment (minotogna) [reply src/client/pages/Geo/GeoMap/GeoMapMenuRecipes/index.ts]: Yes I know you didn't write this component, but let's fix it anyway

File: src/i18n/resources/en.js (hunk 2 above)
Comment (persas) [reply src/i18n/resources/en.js]: I didn't catch what you exactly mean. Will see now in the daily

File: tools/dataMigration/migrateData/_repos.ts [hunk 3]
```diff
@@ -72,4 +72,59 @@ export const isBasicTable = (tableName: string): boolean =>
     'biomassStockTotal',
     'carbonStockAvg',
     'carbonStockTotal',
+    'country_comments_1_1_1',
```
Comment (sorja) [inline tools/dataMigration/migrateData/_repos.ts:75]: can this be moved to 'panEuropeanTables' and extended here like '...panEuropeanTables'

Comment (minotogna) [reply tools/dataMigration/migrateData/_repos.ts:75]: This is just temporary

**Review by sorja — DISMISSED** [review]

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**